elephant/statistics.py

```diff
@@ -240,3 +243,83 @@ def lv(v):
     # calculate LV and return result
     # raise error if input is multi-dimensional
     return 3.*np.mean(np.power(np.diff(v)/(v[:-1] + v[1:]), 2))
+
+
+def psth(sts, w, t_start=None, t_stop=None, output='counts', clip=False):
```

Comment:
Alright, I can make a separate PR. But the PSTH is using the binning class. 
That means, if separated,  it will not pass the tests. If that is acceptable, than fine with me.